clients/src/main/java/org/apache/kafka/common/errors/SerializationException.java

```diff
@@ -41,10 +41,4 @@ public SerializationException() {
         super();
     }
 
-    /* avoid the expensive and useless stack trace for serialization exceptions */
```

Comment:
I'm not convinced it would be that useful to get a stack trace for a `SerializationException` but I don't know for sure that it isn't -- if anyone has any input on this I'm fine with either changing or keeping as is